src/db.rs

```diff
@@ -56,6 +59,59 @@ pub fn get_dbconn() -> mysql::Result<PooledConn> {
     Ok(dbconn)
 }
 
+pub fn db_exists() -> anyhow::Result<()> {
+    let mut dbconn = get_dbconn()?;
+
+    let dbname = env::var("KBS_DB").expect("KBS_DB not set");
```

Comment:
don't we already check this in `get_dbconn`? and speaking of `get_dbconn` if that function returns successfully, don't we already know that the database exists.